### DISCUSSION TITLE
daily_plot.py but yearly instead of daily

### DISCUSSION THREAD

**Discussion by ehpersonal38** [mcguirepr89/BirdNET-Pi#848] [Ideas]
It would be cool to modify the existing `daily_plot.py` to display a "yearly" heatmap of the top 10 detections. Then it would be more akin to the eBird bar charts which I find quite useful:

![chrome_RybpQMl2bK](https://user-images.githubusercontent.com/103586016/231633196-d06d7033-b735-4753-a3be-8f4af5da47f2.png)

**Comment by Svardsten53** [comment]
I really like the daily_plot we're running today, so would love to see it stay. I use a modified version where all today's species are listed. An annual graph, on the other hand, would be a very good addition, preferably as a separate module where you can choose any number of species to be displayed.
